parameterized/parameterized.py

```diff
@@ -386,6 +386,29 @@ def check_input_values(cls, input_values):
             input_values = list(input_values)
         return [ param.from_decorator(p) for p in input_values ]
 
+
+    @classmethod
+    def parameterized_class(cls, properties, test_values):
+        def decorator(base_class):
+            test_class_module = sys.modules[base_class.__module__].__dict__
+            for test_value_key, test_field in enumerate(test_values):
+                test_class_dict = dict(base_class.__dict__)
+
+                if isinstance(properties, string_types):
+                    test_class_dict[properties] = test_field
```

Comment:
This seems like reasonable behavior, but please make sure it's tested and documented